requirements.txt

```diff
@@ -4,14 +4,17 @@ diff-match-patch==20121119
 dj-database-url==0.5.0
 dj-static==0.0.6
 Django==2.0.13
+django-filter==2.0.0
+django-widget-tweaks==1.4.3
 django-debug-toolbar==1.9.1
 django-ical==1.4
 django-recaptcha==1.4.0
 django-registration-redux==2.4
 django-reversion==2.0.13
 django-toolbelt==0.0.1
+django-polymorphic==2.1.2
```

Comment:
Is this being used anywhere? I see it in migrations, but it looks like the fields were later removed

Reply:
This was explored initially to handle the whole "cables are a subset of assets" issue. Has been removed in recent commit